google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -752,6 +753,35 @@ def time hour, minute, second
         end
 
         ##
+        # Creates a new, empty schema instance. This instance can be populated
+        # and passed to {Dataset#load} using the `schema` option. However, for
+        # most use cases, the block yielded by {Dataset#load} is a more
+        # convenient way to configure the schema for the destination table.
+        #
+        # @return [Google::Cloud::Bigquery::Schema]
+        #
+        # @example
+        #   require "google/cloud/bigquery"
+        #
+        #   bigquery = Google::Cloud::Bigquery.new
+        #
+        #   schema = bigquery.schema
+        #   schema.string "first_name", mode: :required
+        #   schema.record "cities_lived", mode: :repeated do |nested_schema|
+        #     nested_schema.string "place", mode: :required
+        #     nested_schema.integer "number_of_years", mode: :required
+        #   end
+        #
+        #   dataset = bigquery.dataset "my_dataset"
+        #
+        #   gs_url = "gs://my-bucket/file-name.csv"
+        #   load_job = dataset.load "my_new_table", gs_url, schema: schema
+        #
+        def schema
```

Comment:
I can do this if you want, although the single call has no importance since there is no API interaction.